src/SystemView.h

```diff
@@ -9,12 +9,17 @@
 #include "graphics/Drawables.h"
 #include "matrix4x4.h"
 #include "vector3.h"
+#include "lua/LuaTable.h"
+#include "lua/LuaPiGui.h"
```

Comment:
If this include isn't required to be in the header file because a class member is using it, please move it to the cpp source file. This helps keep the build (and incremental recompile) time lower!

Reply:
Done! See d9ac37975b2f4161bc1350600ca22d3fe5cedf98